### PR TITLE
Support multiple categories per question

### PR DESCRIPTION
**Pull request by danielnaab**
* Support multiple categories per question
* add /api/v2 for new schema

Implements #887 

Please confirm the following steps are completed:

* [x] Choose the appropriate target branch:
  * Content
    * `preview` (approved content) <- *Content branch*
    * `master` (production) <- `preview`
* [x] Assign an appropriate reviewer:
  * *Content admin* or *Project lead* for merge to `preview` (approved content)
  * *Engineer* for merge to master (production)

:sunglasses: [PREVIEW URL](https://cg-dae9433e-23a0-46d7-bafb-dc79d2d3756c.app.cloud.gov/preview/18f/cv_faq/multiple-categories/)

### CODE REVIEW

**Comment by jonadecker** [comment]
It looks like the three questions per category below the "top questions" block fell off @danielnaab ?

**Comment by jonadecker** [comment]
We probably also need to address the _includes/meta.html canonical treatment, which is going to create a canonical for each category if I'm reading it correctly @danielnaab ?

**Comment by jonadecker** [comment]
Something new and interesting occurring @danielnaab  

From the home page, clicking on any link to a question (not a category) results in a breadcrumb of FAQ > Animals > ...

E.g., Top questions, click on "How does it spread?" - you land on the question, and the URL is /spread/ but the breadcrumb is Animals

Screen Shot 2020-07-16 at 1 17 43 PM

This isn't true of any of the other Top questions because they're linking to anchors within category pages. But the direct links to FAQs below Top questions show the same breadcrumb.

**Comment by danielnaab** [comment]
> From the home page, clicking on any link to a question (not a category) results in a breadcrumb of FAQ > Animals > ...

Good catch - should be fixed now.

**Comment by debjudy** [comment]
@danielnaab If a question has "promoted: true" and is in multiple categories, is it then promoted in all categories?
 The animal example (_content/animals/are-pets-from-a-shelter-safe.md) is a good one.  We might want it promoted in the animals category but not in spread.

**Comment by danielnaab** [comment]
> @danielnaab If a question has "promoted: true" and is in multiple categories, is it then promoted in all categories?
> The animal example (_content/animals/are-pets-from-a-shelter-safe.md) is a good one. We might want it promoted in the animals category but not in spread.

Yes, with this implementation it's promoted in all categories - will require additional work if we want to handle that condition.

File: api/v2/openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+openapi: 3.0.0
+
+info:
+  title: "Coronavirus FAQ API"
+  version: "1.0.0"
```
Comment (danielnaab) [inline api/v2/openapi.yaml:5]:
```suggestion
  version: "2.0.0"
```

**Pull request merged**